nomos-blend/message/src/encap/decapsulated.rs

```diff
@@ -21,8 +23,14 @@ pub enum DecapsulationOutput<const ENCAPSULATION_COUNT: usize> {
     reason = "Size difference between variants is not too large (small ENCAPSULATION_COUNT)"
 )]
 pub(super) enum PartDecapsulationOutput<const ENCAPSULATION_COUNT: usize> {
-    Incompleted((EncapsulatedPart<ENCAPSULATION_COUNT>, PublicHeader)),
-    Completed(Payload),
+    Incompleted(
```

Comment:
I think it's the right time to introduce a struct for this 🙈

Reply:
Maybe for all enums. What do you think? Can be a separate PR or we can just not do it, tho.

Reply:
Yeah, I thought about that too. Applied in 7bbf5fb7acbd0a98d2bded6644bbb3604e5330b1